src/Flicking.js

```diff
@@ -1743,6 +1756,38 @@ export default class Flicking extends Mixin(Component).with(eventHandler) {
 		return this;
 	}
 
+	/**
+	 * Rebuild/Initialize panels by current DOM of panels.
+	 * @ko 현재 패널 DOM 을 기준으로 패널을 재구성/초기화한다.
+	 * @method eg.Flicking#rebuild
+	 * @example
+	 * 		flicking.rebuild();
+	 *
+	 * @return {eg.Flicking} An instance of a module itself <ko>모듈 자신의 인스턴스</ko>
+	 */
+	rebuild() {
+		const panel = this._conf.panel;
+		const options = this.options;
+		const prefix = options.prefix;
+
+		// filter original panel (remove clones)
+		utils.toArray(this.$wrapper.querySelectorAll(`.${prefix}-clone`)).forEach(el => el.remove());
+		panel.$list = utils.toArray(this.$container.children);
+
+		// panels' css values
+		this._initOriginalPanelStyle(panel.$list);
+
+		// Add clones
+		if (this._addClonePanels()) {
+			panel.$list = utils.toArray(this.$container.children);
+		}
+
+		this._setDefaultPanel(options.defaultIndex);
```

Comment:
@jongmoon 
Shouldn't `defaultIndex` be the current index when calling `rebuild`?

Reply:
@daybrush I think `rebuild` is used for working like initial time. Maintaining current index is policy issue. How about to add parameter?

```js
flicking.rebuild({defaultIndex: 1});
```

Reply:
@jongmoon 

When I heard, I knew that the contents of the panel were updated in the `circular`.

Reply:
@jongmoon

I think adding parameters is good.